asv_bench/benchmarks/pandas/timeseries.py

```diff
@@ -0,0 +1,28 @@
+from datetime import timedelta
+
+import dateutil
+import numpy as np
+
+from modin.pandas import (
+    DataFrame,
+    Series,
+    date_range,
+    period_range,
+    timedelta_range,
+)
+
+
+class ResetIndex:
```

Comment:
We already have benchmarks for `reset_index`. Can be safely removed.

Reply:
removed.